os/mm/mm_heap/mm_getheap.c

```diff
@@ -76,12 +76,13 @@ void mm_add_app_heap_list(struct mm_heap_s *heap, char *app_name)
 		node = dq_next(node);
 	}
 
-	node = (app_heap_s *)kmm_malloc(sizeof(app_heap_s));
+	node = (app_heap_s *)kmm_zalloc(sizeof(app_heap_s));
```

Comment:
True. This is not required now since I am setting is_active variable. I had modified it initially to check the memory leak issue. I reverted this change now.